clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -41,8 +41,17 @@
 {{$ENABLE_RESTART_COUNT_CHECK := DefaultParam .ENABLE_RESTART_COUNT_CHECK false}}
 {{$RESTART_COUNT_THRESHOLD_OVERRIDES:= DefaultParam .RESTART_COUNT_THRESHOLD_OVERRIDES ""}}
 
+# Check if custom namespace prefix is set
+{{$NAMESPACE_PREFIX := DefaultParam .CL2_NAMESPACE_PREFIX ""}}
+
 name: access-tokens
 namespace:
+{{if $NAMESPACE_PREFIX}}
```

Comment:
I'm not 100% convinced that this should be guarded by NAMESPACE_PREFIX - intuitively if I set namespace-prefix, I dn't expect  those other parameters to be set for me...

Reply:
1. I agree that there could be a better name about it.
2. There are two configuration that we are interested in - default testing, that we are already using, with deleteStaleNamespace/deleteAutomanagedNamespaces/enableExistingNamespaces being true/true/false, and partial testing, that uses false/false/true.

Solution: rename to CL2_RUN_PARTIAL_TEST_IN_NAMESPACE?

Reply:
Yeah - that name sounds better.
@mm4tt - or you maybe have better ideas how to make the config simpler?

Reply:
Hmm, ideally one could simply re-use the test config as is and specify namespace setting via flags, but it looks like we abandoned this approach. 

Now, the only thing that comes to my mind is to get rid of the `if` block and instead introduce variables for `deleteStaleNamespaces`, `deleteAutomanagedNamespaces`, etc. with proper defaults.  That way, Kuba can run his test with all these envs overridden to achieve what he needs.

If you're going to go with what you currently have, then +1 to naming it in a more self-documenting way.